docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  home-assistant:
+    image: homeassistant/home-assistant
```

Comment:
Minor suggestion: I would suggest using the `homeassistant/home-assistant:latest` tag. It makes it clearer how to pin a version